sdk/typescript/test/e2e/signature-mpc.test.ts

```diff
@@ -133,7 +127,7 @@ describe('Test key share transfer', () => {
 		);
 
 		let secretUserShare = new Uint8Array(256);
-		secretUserShare.set(Uint8Array.from(Buffer.from(secretKeyshare, 'hex')).reverse());
+		secretUserShare.set(secretShare.reverse());
```

Comment:
why do you need the reverse?

Reply:
idk tbh it was in the original code @yaelAbergel